Simplified/NYPLBookRegistry.m

```diff
@@ -300,14 +300,6 @@ - (void)syncWithStandardAlertsOnCompletion
 {
   [self syncWithCompletionHandler:^(BOOL success) {
     if(success) {
-      [[[UIAlertView alloc]
-        initWithTitle:NSLocalizedString(@"SyncComplete", nil)
-        message:NSLocalizedString(@"YourBooksWereSyncedSuccessfully", nil)
-        delegate:nil
-        cancelButtonTitle:nil
-        otherButtonTitles:NSLocalizedString(@"OK", nil), nil]
-       show];
-      
```

Comment:
instead of removing the alert here, you might want to use this method with a `syncWithCompletionHandler:nil`